torch/distributed/algorithms/quantization/quantization.py

```diff
@@ -38,7 +41,7 @@ def _quantize_tensor_list(tensor_list, qtype):
         raise RuntimeError(
             f"_quantize_tensor_list expecting list of torch.Tensor as input but found {type(tensor_list)}"
         )
-    if (qtype == DQuantType.FP16):
+    if ((qtype == DQuantType.FP16) or (qtype == DQuantType.BFP16)):
```

Comment:
for `_quantize_tensor_list`, should we not do the check because it's doing the same check as `_quantize_tensor`? we can just leverage `_quantize_tensor` to throw error if `qtype` is not support?